src/components/footer/Footer.jsx

```diff
@@ -20,14 +22,14 @@ const Footer = () => {
   return (
     <footer className={footerStyles.footer}>
       <div className={footerStyles.footerContent}>
-        <a href="https://github.com/JuliaMV/culture-portal" target="_blank" rel="noopener noreferrer" title="link to GitHub project" alt="link to GitHub project">
-          <Img fixed={data.file.childImageSharp.fixed} className={footerStyles.footerImg} />
+        <a href="https://github.com/JuliaMV/culture-portal" target="_blank" rel="noopener noreferrer" title="link to GitHub project">
+          <Img fixed={data.file.childImageSharp.fixed} className={footerStyles.footerImg} alt="link to GitHub project" />
         </a>
-        <p className={footerStyles.footerText}>
-          Created by
+        <div className={footerStyles.footerText}>
+          <FormattedMessage id="footerMessage" />
           <span className={footerStyles.styleSpan}> GROUP-19</span>
```

Comment:
i think we need remove first whitespace: `<span className={footerStyles.styleSpan}>GROUP-19</span>`

Reply:
Then we will have to add it in the message, otherwise the text will be glued.